smart_campus/app/management/commands/setroles.py

```diff
@@ -0,0 +1,9 @@
+from django.core.management.base import BaseCommand, CommandError
+from app.models import Role
```

Comment:
One blank line above.
Separate std libs, 3rd party libs and your own packages is a good practice.

Reply:
Got it